### PR TITLE
add_edges correct docstring parameters

### LINKED ISSUE #1178: display.add_edeges uncorrect docstring ?
dcostring says inputs are the map and the affine 
```Python
def add_edges(self, img, color='r'):
    """ Plot the edges of a 3D map in all the views.
    Parameters
    ----------
    map: 3D ndarray
    affine: 4x4 ndarray
```
but function takes only a Niimg-like object :
```Python
import nibabel
from nilearn import datasets, plotting
icbm152 = datasets.fetch_icbm152_2009()
img = nibabel.load(icbm152.gm)
display = plotting.plot_anat(icbm152.t1)
display.add_edges(img.get_data(), img.affine)
```
gives an error

### PR DESCRIPTION
**Pull request by salma1601**
fixes #1178

### CODE REVIEW

**Comment by bthirion** [comment]
LGTM. CircelCI failure seems unrelated.

File: nilearn/plotting/displays.py [hunk 1]
```diff
@@ -746,12 +746,9 @@ def add_edges(self, img, color='r'):
...
         ----------
-        map: 3D ndarray
-            The 3D map to be plotted. If it is a masked array, only
-            the non-masked part will be plotted.
-        affine: 4x4 ndarray
-            The affine matrix giving the transformation from voxel
-            indices to world space.
+        img: Niimg-like object
+            See http://nilearn.github.io/manipulating_images/input_output.html.
+            Provides image to plot.
```
Comment (KamalakerDadi) [inline nilearn/plotting/displays.py]: May be we should adapt well to the same conventions as was before. You now know better what to write following from issue. :)

"The 3D map to be plotted. If it is a masked array, only the non-masked part will be plotted"

**Comment by KamalakerDadi** [comment]
Small comment from my side.

File: nilearn/plotting/displays.py [hunk 2]
```diff
@@ -746,12 +746,10 @@ def add_edges(self, img, color='r'):
...
-        map: 3D ndarray
-            The 3D map to be plotted. If it is a masked array, only
-            the non-masked part will be plotted.
-        affine: 4x4 ndarray
-            The affine matrix giving the transformation from voxel
-            indices to world space.
+        img: Niimg-like object
+            See http://nilearn.github.io/manipulating_images/input_output.html.
+            The 3D map to be plotted.
+            If it is a masked array, only the non-masked part will be plotted.
```
Comment (KamalakerDadi) [inline nilearn/plotting/displays.py]: Sorry for not being clear. I expected something like this,

"The 3D image to be plotted. If it is masked, only the non-masked part will be plotted/projected."

Comment (KamalakerDadi) [inline nilearn/plotting/displays.py]: Also, I will take doc to top and See link to bottom.

Comment (AlexandreAbraham) [inline nilearn/plotting/displays.py]: > Also, I will take doc to top and See link to bottom.

In all the nilearn docstrings, the link comes first.

**Comment by AlexandreAbraham** [comment]
Thanks, merging.

File: nilearn/plotting/displays.py (hunk 2 above)
Comment (KamalakerDadi) [inline nilearn/plotting/displays.py]: >In all the nilearn docstrings, the link comes first.

Yes, I noticed that.

**Pull request merged**